models/order.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+
+var orderSchema = new mongoose.Schema({
+  status: String
```

Comment:
yes def i just made it as a place holder for now. will work on it some more tonight.